app/scripts/metamask-controller.js

```diff
@@ -1701,7 +1698,6 @@ export default class MetamaskController extends EventEmitter {
       initState.SmartTransactionsController,
     );
 
-    ///: BEGIN:ONLY_INCLUDE_IF(petnames)
```

Comment:
This fence around the `NameController` and address book bridge has reminded me that we didn't fence the original address book migration meaning some users could have petnames waiting for them once this code fence is gone.

But the bridge won't have been running in the meantime since that was in the code fence.

Should we then duplicate the address book migration (100) to run again for the next version to synchronise with any address book changes in the interim?

Reply:
Yes I'll add it. 

And we would have had to do the same thing with the account id => petnames migration had we merged that PR before this one.

Reply:
Done.